workspace/spec/lita/handlers/teamcity_spec_anki.rb

```diff
@@ -154,4 +154,47 @@
       expect(output_string).to eq(expected_output)
     end
   end
+
+  describe '#fetch_queue_build' do
+    queue_build_json_str = '{"count": 2,
+                             "build": [{"buildTypeId": "CozmoOne_MasterBuild",
+                                        "branchName": "BI-123-XYZ",
+                                        "id": "4307"},
+                                       {"buildTypeId": "OverDriveOne_Release_Build",
+                                        "branchName": "BI-124-XYZ",
+                                        "id": "4308"}]}'
+    before do
+      stub_request(:get, /buildQueue/).to_return(:status => 200, :body => queue_build_json_str)
+    end
+    it 'buildTypeId is found that returns with the queue build correctly' do
```

Comment:
please make the message easier to understand. for exp: `returns correct queue build if buildTypeId is found`. Also update for all other description for the consistence.